authz/rbac_translator.go

```diff
@@ -305,10 +305,10 @@ func (options *auditLoggingOptions) toProtos() (allow *v3rbacpb.RBAC_AuditLoggin
 	for i := range options.AuditLoggers {
 		config := &options.AuditLoggers[i]
 		if config.Config == nil {
-			return nil, nil, fmt.Errorf("AuditLogger Config field cannot be nil")
+			config.Config = &structpb.Struct{}
```

Comment:
I suggest you change this field back to a value so that no nil-check is needed and `config.Config` will default to an empty struct.

Reply:
Before I go through and change it, I want to triple check it's what we want.
IIRC we've flipped this back and forth already.
We _definitely_ are fine with saying `nil` and `{}` are the same? So we make this field a value instead of a pointer, then if it's missing it just defaults to the empty structpb.Struct?

We changed it in the first place to differentiate between nil and empty, right?

Reply:
Fixed, PTAL